lib/patches/stdlib/rubygems.rb

```diff
@@ -1,7 +1,7 @@
 require 'openssl-stubs' if Truffle::Boot.patching_openssl_enabled?
 
 Truffle::Patching.require_original __FILE__
-Truffle::Patching.install_gem_activation_hook if Truffle::Boot.get_option 'patching'
+Truffle::Patching.install_gem_activation_hook
```

Comment:
Sorry I meant: Why is the condition removed here?

Reply:
See a778da3cef07878113c5f7eb20116d899ec32bc5:
> This file is only required if patching is enabled.

Reply:
ah, ok then.